apps/src/music/views/MiniMusicPlayer.tsx

```diff
@@ -81,25 +89,55 @@ const MiniPlayerView: React.FunctionComponent<MiniPlayerViewProps> = ({
 
     // Play sounds
     playerRef.current.playSong(sequencerRef.current.getPlaybackEvents());
+    setCurrentProjectId(project.id);
+  }, []);
+
+  const onStopSong = useCallback(async () => {
+    playerRef.current.stopSong();
+    setCurrentProjectId(undefined);
   }, []);
 
   // Some loading UI while we're fetching the library
   if (isLoading) {
     return <div>Loading...</div>;
   }
 
-  // Bare bones render function to hook things together
   return (
-    <div>
+    <div className={moduleStyles.miniMusicPlayer}>
       {projects.map(project => {
         return (
-          <button
-            type="button"
+          <div
+            className={moduleStyles.entry}
             key={project.id}
-            onClick={() => onPlaySong(project)}
+            onClick={() => {
+              project.id === currentProjectId
+                ? onStopSong()
+                : onPlaySong(project);
+            }}
           >
-            Play! {project.name}
-          </button>
+            <div className={moduleStyles.control}>
+              <FontAwesomeV6Icon
+                iconName={project.id === currentProjectId ? 'stop' : 'play'}
+                iconStyle="solid"
+                className={moduleStyles.icon}
+              />
+            </div>
+            <div className={moduleStyles.name}>{project.name}</div>
+            <a
```

Comment:
could potentially use the design system `Link.tsx` here (if it works with these styles)

Reply:
Oh, good call.  Will do in follow-up.